lib/openhab/dsl/rules/builder.rb

```diff
@@ -1514,7 +1514,12 @@ def thing_updated(attach: nil)
         #
         def event(topic, source: nil, types: nil, attach: nil)
           types = types.join(",") if types.is_a?(Enumerable)
-          trigger("core.GenericEventTrigger", eventTopic: topic, eventSource: source, eventTypes: types, attach: attach)
+          trigger("core.GenericEventTrigger",
```

Comment:
maybe add a comment here that the duplication is because the parameters were renamed in 4.0.

actually, we should come up with a consistent comment so that when we drop 3.4 in the future, it's easy to find the code that needs removed.

Reply:
how about it now?